### DISCUSSION TITLE
Persistent notification

### DISCUSSION THREAD

**Discussion by MaxR8684** [Blackymas/NSPanel_HA_Blueprint#2271] [Q&A]
Hi there, Is there a way to keep a notification visible on screen and the screen turned on until it is confirmed via the button?

**Comment by edwardtfn** [comment]
You can add as a feature request... 😉

**Comment by MaxR8684** [comment]
I solved this simple with an automation the display sleep time changes to 5 hours wenn the screen state changes from screensaver to notification but i stil have the issue that the notification doesnt stay on screen for longer than 15 minutes.

Is there a simple way to get this to stay longer?

Or is this a Feature request thing?

**Reply by edwardtfn** [reply]
It should be a feature request... Probably an easy one to implement...

**Comment by MaxR8684** [comment]
I think it allready works, last time my son did Hit te x button instead of confirming the notification.Layer 8 issue.
-------- Ursprüngliche Nachricht --------Von: Edward Firmo ***@***.***> Datum: 16.09.24  13:47  (GMT+01:00) An: Blackymas/NSPanel_HA_Blueprint ***@***.***> Cc: Max Rudolf ***@***.***>, State change ***@***.***> Betreff: Re: [Blackymas/NSPanel_HA_Blueprint] Persistent notification
  (Discussion #2271) 
It should be a feature request... Probably an easy one to implement...
